cmd/pg/copy.go

```diff
@@ -62,9 +62,4 @@ func init() {
 		withoutHistoryShorthand,
 		false,
 		withoutHistoryDescription)
-
-	_ = backupCopyCmd.MarkFlagFilename(toConfigFile)
```

Comment:
Why remove this?

Reply:
1. These calls don't affect anything. Both `MarkFlagFilename` and `MarkFlagRequired` require flag name but flag value was provided. Just a bug I guess
2. `MarkFlagFilename` call without extension arg is pointless
3. Seems like toFlag and fromFlag actually aren't required. `ReadConfigFromFile` handles empty `configFile` argument

Reply:
Well, wal-g copy command definetly requires both `fromFlag` and `toFlag` to be specified. So I don't see any reason to remove this logic. Let's just fix the incorrect argument, e.g. change it to this:
```
	_ = backupCopyCmd.MarkFlagRequired(toFlag)
	_ = backupCopyCmd.MarkFlagRequired(fromFlag)
```